app/src/main/res/values/strings.xml

```diff
@@ -810,16 +810,16 @@
     <string name="security">Security</string>
     <string name="set_mode_for_app_ops_dots">Set mode for app ops…</string>
     <string name="backup_source_description">Back up the entire source directory.</string>
-    <string name="backup_apk_only_description">Back up only the APK files from the source directory. <b>Source</b> must be selected for this to work.</string>
-    <string name="backup_data_description">Back up data folders. If selected, internal data folders will be backed up by default.</string>
-    <string name="backup_external_data_description">Back up external data folders. <b>Data</b> must be selected for this to work.</string>
+    <string name="backup_apk_only_description">Only back up the APK files from the source directory you select.</string>
```

Comment:
“You select” has a different meaning. Since this goes in the subtitle section, you don't have to make it concise, rather it's expected to be as descriptive as possible.

Reply:
I couldn't tell you why it is like that. Probably meant to do something, but can't tell what. Is "<b>Source</b> must be selected for this to work" important info?

Reply:
> "Source must be selected for this to work" important info?

It is. This option has no effect without it. Maybe you can write the whole thing in a complex/compound sentence.